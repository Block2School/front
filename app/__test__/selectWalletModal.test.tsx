import renderer from 'react-test-renderer'
import React from 'react'
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import { shallow } from 'enzyme';
import { render, screen } from '@testing-library/react';
import TutorialCategoryCard from '../components/cards/tutorialCategoryCard';
import SelectWalletModal from '../components/modals/wallets/walletsModal';


describe('<SelectWalletModal/>', () => {
  let wrapper: any;
  test('should render correctly', () => {
    wrapper = shallow(<SelectWalletModal isOpen={false} closeModal={undefined}/>)
  });

  test('match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  }) 
})