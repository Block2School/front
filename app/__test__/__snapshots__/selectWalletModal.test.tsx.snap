// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectWalletModal/> match snapshot 1`] = `ShallowWrapper {}`;
